discord/interactions/connectCommand.ts

```diff
@@ -29,33 +48,132 @@ export const handleConnectCommand = async (
     return;
   }
 
-  const alreadyDiscordMember = await DiscordMemberRepository.findOneBy({
+  const alreadyDiscordMember = await DiscordMemberRepository.findBy({
```

Comment:
please use plural here
```suggestion
  const alreadyDiscordMembers = await DiscordMemberRepository.findBy({
```

Reply:
hey @MohammedChakor please use plural as I suggested here :) We may be handling multiple discord members (it's a list) so name the variable `alreadyDiscordMembers`